intellij-plugin-structure/structure-base/src/main/kotlin/com/jetbrains/plugin/structure/base/utils/CompatibilityUtils.kt

```diff
@@ -0,0 +1,61 @@
+package com.jetbrains.plugin.structure.base.utils
+
+import java.util.ArrayList
+
+
+class CompatibilityUtils {
+  companion object {
+    private const val MAX_BRANCH_VALUE = 1000
+    private const val MAX_COMPONENT_VALUE = 10000
+    private const val MAX_BUILD_VALUE = 100000
+    private val NUMBERS_OF_NINES by lazy { initNumberOfNines() }
+
+    fun versionAsLong(vararg components: Int): Long {
```

Comment:
I believe Ktor version is always represented by 3 integer numbers: `X.Y.Z`, hence no  `if (components.size >= 3)` is needed and `vararg components` can be just 3 integer arguments

Reply:
This is common method that we're using not only for Ktor, but for Idea plugins too. That's why we allow `components.size != 3`. To validate ktor version components we have `KtorVersion.fromString` method.